Drv/Ip/TcpClientSocket.cpp

```diff
@@ -57,6 +57,11 @@ SocketIpStatus TcpClientSocket::openProtocol(NATIVE_INT_TYPE& fd) {
 #if defined TGT_OS_TYPE_VXWORKS || TGT_OS_TYPE_DARWIN
     address.sin_len = static_cast<U8>(sizeof(struct sockaddr_in));
 #endif
+    // Enable Address reuse to avoid binding to the socket that still might be in TIME_WAIT state.
+    // Can happen if a function like get_free_port() is called to determine an available port by binding to port 0.
+    const int enable = 1;
+    if (setsockopt(socketFd, SOL_SOCKET, SO_REUSEADDR, &enable, sizeof(enable)) < 0)
```

Comment:
Minor coding-standard note:  all `if` statements regardless of line count should contain explicit block curly braces. i.e.

```c++
if (something) {

}
```

This is done to prevent bug that might creep-in if ever a step is added to this if block on a future revision.

Reply:
Totally agree. I thought it was already identified by clang-tidy, but it's currently not.

Would you be in to have clang-tidy verifying that there is always a braces after statements? I can prepare a PR for that.

Reply:
yes, absolutely!